where-for-dinner/doc/TAPDeployment.md

```diff
@@ -185,19 +198,17 @@ The accelerator contains the following configuration options:
-* **Create Resource Claim Definition for Service:** If this box is checked, the accelerator will generate configuration yaml file in the `config/app-operator` directory that contain the resource definition for the resource claims that the micro-services can use to create service bindings to external service like RabbitMQ and database.  It also creates resource definitions used by the Tanzu CLI to manage service instances and resource claims. 
-* **Enable Security:** If this box is checked, the accelerator configure the applications to use a `secure` profile that will require the UI application authenticate users and for micro-services to require valid oAuth tokens with each request.  The accelerator will also generate a file named `appSSOInstance.yaml` in the `config/service-operator` directory that contains the configuration to create an AppSSO authorization server.  It will also generate a file named `clientRegistrationResourceClaim.yaml` in the `config/app-operator` directory that contains configuration for creating a ClientRegistration resource as well as the resource claims that the micro-services can use to create service bindings to AppSSO instance (via the ClientRegistration).  **NOTE:**  Security is not completely functional at this time.
-* **AppSSO Instance Name:**  If security is enabled, the name of the AppSSO resource that the micro-services will connect to.  This name will also be propagated to the `workload.yaml` files in the resource claim section to indicate the names of the AppSOO resource that micro-services should connect to.
-* **Issuer URI:**  If security is enabled, this is the full issuer URI used by the AppSSO instance.  This URI will also be used to create `Service` and `HTTPProxy` resources for Ingress to the AppSSO instance.  Make sure the URI is routed to your cluster.  You will likely need to register the host name of the issuer URI with your DNS provider to route to your cluster's Ingress load balancer.
-* **Redirect URI:**  If security is enabled, this is the redirect URI back the API gateway service after a successful user authentication.
+* **Database Name:**  The name of the database resource that the micro-services will connect to.  This name will be propagated to the `workload.yaml` files in the resource claim section to indicate the names of the database that micro-services should connect to.
+* **Enable Cloud Events:** If this box is checked, the accelerator will generate a configuration yaml file in the `config/app-operator` directory that contains the resource definitions for the KNative eventing resources.  These include the RabbitMQ source as well as the broker and trigger configurations for routing events to downstream services.  This will also configure applicable services to remove the Spring Cloud Stream binding libraries from the build process.
+* **Use RabbitMQ KNative Eventing Broker:** If this box is checked, the KNative broker will use a RabbitMQ broker implementation.
```

Comment:
KNative should be Knative

Reply:
The catalog items may need to fixed for 1.3.1 as w. ll. The catalog updates that were done for 1.3.0 renamed the references to the directories to "where-for-dinner" but did not actually change the directory names.

Reply:
Catalog file and directory names have been updated in this PR.